pkg/controllermanager/controller/plant/plant_utils.go

```diff
@@ -0,0 +1,106 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plant
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	gardencorev1alpha1 "github.com/gardener/gardener/pkg/apis/core/v1alpha1"
+	"github.com/gardener/gardener/pkg/apis/core/v1alpha1/helper"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/discovery"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// Following labels come from k8s.io/kubernetes/pkg/kubelet/apis
+
+	// LabelZoneRegion zone Region label
+	LabelZoneRegion = "failure-domain.beta.kubernetes.io/region"
```

Comment:
Meanwhile, we have vendored Kubernetes' `release-1.14` branches. You can remove this and use `https://github.com/kubernetes/api/blob/release-1.14/core/v1/well_known_labels.go#L22` instead.